From 4bef27d85c6e03cf160ab75a25df837b6e5927c2 Mon Sep 17 00:00:00 2001
From: Adam Williamson <awilliam@redhat.com>
Date: Thu, 23 Jan 2025 16:28:04 -0800
Subject: [PATCH 2/2] data: Correct Utilities category ordering, add comments

This adds a comment to make clear that the Utilities category is
intended to be organized alphabetically by the display name of
the apps (*not* by the desktop file names), and adds comments
with the current display name of each app.

Signed-off-by: Adam Williamson <awilliam@redhat.com>
Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/3609>
---
 js/ui/appDisplay.js | 33 +++++++++++++++++----------------
 1 file changed, 17 insertions(+), 16 deletions(-)

diff --git a/js/ui/appDisplay.js b/js/ui/appDisplay.js
index 56f3cca4b..55f4fa0d2 100644
--- a/js/ui/appDisplay.js
+++ b/js/ui/appDisplay.js
@@ -60,22 +60,23 @@ const DEFAULT_FOLDERS = {
     'Utilities': {
         name: 'X-GNOME-Shell-Utilities.directory',
         apps: [
-            'org.freedesktop.GnomeAbrt.desktop',
-            'nm-connection-editor.desktop',
-            'org.gnome.baobab.desktop',
-            'org.gnome.Characters.desktop',
-            'org.gnome.Connections.desktop',
-            'org.gnome.DejaDup.desktop',
-            'org.gnome.DiskUtility.desktop',
-            'org.gnome.Evince.desktop',
-            'org.gnome.FileRoller.desktop',
-            'org.gnome.font-viewer.desktop',
-            'org.gnome.Logs.desktop',
-            'org.gnome.Loupe.desktop',
-            'org.freedesktop.MalcontentControl.desktop',
-            'org.gnome.seahorse.Application.desktop',
-            'org.gnome.tweaks.desktop',
-            'org.gnome.Usage.desktop',
+            // Sorted by name as shown in menus, not filename
+            'nm-connection-editor.desktop', // Advanced Network Configuration
+            'org.gnome.DejaDup.desktop', // Backups
+            'org.gnome.Characters.desktop', // Characters
+            'org.gnome.Connections.desktop', // Connections
+            'org.gnome.DiskUtility.desktop', // Disks
+            'org.gnome.baobab.desktop', // Disk Usage Analyzer
+            'org.gnome.Evince.desktop', // Document Viewer
+            'org.gnome.FileRoller.desktop', // File Roller
+            'org.gnome.font-viewer.desktop', // Fonts
+            'org.gnome.Loupe.desktop', // Image Viewer
+            'org.gnome.Logs.desktop', // Logs
+            'org.freedesktop.MalcontentControl.desktop', // Parental Controls
+            'org.gnome.seahorse.Application.desktop', // Passwords and Keys
+            'org.freedesktop.GnomeAbrt.desktop', // Problem Reporting
+            'org.gnome.tweaks.desktop', // Tweaks
+            'org.gnome.Usage.desktop', // Usage
         ],
     },
     'YaST': {
-- 
2.47.1

